src/QMCHamiltonians/tests/test_QMCHamiltonian.cpp

```diff
@@ -203,6 +204,8 @@ TEST_CASE("integrateListeners", "[hamiltonian]")
 
   QMCHamiltonian::mw_evaluate(ham_list, twf_list, p_list);
```

Comment:
this call also returns a std::vector of local energy which should match. 

And the sum of the per particle component energies and the sum of local energies per particle sum should match this returned value as well. (This is a test I should have added so you can just drop an issue for me.)

Reply:
> this call also returns a std::vector of local energy which should match.

I added checks for this.

> And the sum of the per particle component energies and the sum of local energies per particle sum should match this returned value as well. (This is a test I should have added so you can just drop an issue for me.)

Added https://github.com/QMCPACK/qmcpack/issues/4769